releasenotes/notes/fix-unitary-synth-1q-circuit-756ad4ed209a313f.yaml

```diff
@@ -0,0 +1,6 @@
+---
+fixes:
+  - |
+    Fixed an issue with the :class:`~.UnitarySynthesis` pass where a circuit
+    with 1 qubit gates and a :class:`~.Target` input would sometimes fail
+    instead of processing the circuit as expected.
```

Comment:
We should probably specify that it's just the default plugin that was borked - things like `aqc` could presumably still have worked.

Reply:
Heh, well that's why I put "sometimes" :P but yeah mention it was the default plugin would have been more precise. We can update this in the release note cleanup PR